CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.25.0](https://github.com/rudderlabs/rudder-sdk-js/compare/v2.23.0...v2.25.0) (2023-02-20)
```

Comment:
This should have compared 2.24.0 to 2.25.0

### Bug Fixes seems to be in the diff of 2.23.0 to 2.24.0 and is duplicated

Reply:
Adressed